x-pack/plugins/canvas/public/expression_types/arg_types/series_style/extended_template.js

```diff
@@ -37,13 +38,29 @@ export const ExtendedTemplate = props => {
     { value: 5, text: '5' },
   ];
 
-  const labelOptions = [{ value: '', text: 'Select Series' }];
+  const labelOptions = [
+    {
+      value: '',
+      text: intl.formatMessage({
+        id: 'xpack.canvas.expressionTypes.seriesStyle.selectSeriesLabel',
```

Comment:
xpack.canvas.expressionTypes.seriesStyle => xpack.canvas.seriesStyle for all ids in this file